extensions-c9s.yaml

```diff
@@ -74,3 +74,9 @@ extensions:
       - sig-virtualization
     packages:
       - kata-containers
+  # https://issues.redhat.com/browse/OCPSTRAT-649
+  kernel-64k-pages:
```

Comment:
Naming bikeshed: is it really worth "renaming" the kernel package here?  i.e. vs just calling it `kernel-64k`